### PR TITLE
Add SectionTitle component to global components

### PR DESCRIPTION
**Pull request by Tetheta**
Added it as a placeholder in Home as well, will edit a bit to try
out more things and put a welcome message next time

### CODE REVIEW

**Comment by Tetheta** [comment]
image

File: Ash.FrontEnd/src/global/style.less [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.ash-section {
+  margin: 10px;
```
Comment (blakestone95) [inline Ash.FrontEnd/src/global/style.less]: New design will have these sections span the whole width of their containers, so it doesn't need left and right margins.

File: Ash.FrontEnd/src/global/style.less [hunk 2]
```diff
@@ -0,0 +1,20 @@
+.ash-section {
+  margin: 10px;
+  border-radius: 10px;
+  background-color: #333;
```
Comment (blakestone95) [inline Ash.FrontEnd/src/global/style.less]: Create a less variable for this, please!

File: Ash.FrontEnd/src/global/style.less [hunk 3]
```diff
@@ -0,0 +1,20 @@
+.ash-section {
+  margin: 10px;
+  border-radius: 10px;
+  background-color: #333;
+
+  div {
+    padding: 20px 20px 10px;
```
Comment (blakestone95) [inline Ash.FrontEnd/src/global/style.less]: Imo, 3-arg padding/margin rules are confusing cause I can never remember which rule is copied to the opposite side.  Do you mind making this 4-arg?

File: Ash.FrontEnd/src/global/style.less [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  border-radius: 10px;
+  background-color: #333;
+
+  div {
+    padding: 20px 20px 10px;
+
+    h1 {
+      margin: 0;
+      font-size: 36px;
+      font-weight: 380;
```
Comment (blakestone95) [inline Ash.FrontEnd/src/global/style.less]: I didn't realize you could specify values between the 100's divisions lol!

**Comment by blakestone95** [comment]
Looks awesome, btw!  Just a few adjustments.

File: Ash.FrontEnd/src/global/style.less (hunk 2 above)
Comment (Tetheta) [reply Ash.FrontEnd/src/global/style.less]: Will do

File: Ash.FrontEnd/src/global/style.less (hunk 3 above)
Comment (Tetheta) [reply Ash.FrontEnd/src/global/style.less]: Can if you want, Google has the opposite rule haha hence my default. I'll change it

File: Ash.FrontEnd/src/global/style.less (hunk 4 above)
Comment (Tetheta) [reply Ash.FrontEnd/src/global/style.less]: Lol yeah, though think my zoom was just being weird, I should probably change to 400

**Comment by Tetheta** [comment]
I'll make those adjustments tomorrow if I can

**Pull request merged**